packages/react/src/CTABanner/CTABanner.module.css

```
@@ -0,0 +1,82 @@
+.CTABanner {
+  position: relative;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+}
+
+.CTABanner--Shadow {
+  position: absolute;
+  transform: translate(-50%, 0%);
+  width: calc(60% + 60px);
+  height: calc(100% + 60px);
+  border-radius: 50% 22% 40% 80%;
+  filter: blur(50px);
+  top: 0%;
+  opacity: 0.7;
+}
+
+.CTABanner--StartShadow {
+  left: 30%;
+  background: radial-gradient(
+    circle at 50% 50%,
+    hsla(var(--brand-CTABanner-shadow-color-start) / 1),
+    hsla(var(--brand-CTABanner-shadow-color-start) / 0)
+  );
+}
+
+.CTABanner--EndShadow {
+  left: 70%;
+  background: radial-gradient(
+    circle at 50% 50%,
+    hsla(var(--brand-CTABanner-shadow-color-end) / 1),
+    hsla(var(--brand-CTABanner-shadow-color-end) / 0)
+  );
+}
+
+.CTABanner-content {
+  border-radius: var(--brand-borderRadius-medium);
+  padding: var(--base-size-40) var(--base-size-32);
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  gap: var(--base-size-16);
+  background: white;
```

Don't think this will work on dark mode..

```suggestion
  background: var(--brand-color-canvas-default);
```